low-level/framework/utils/utility.py

```diff
@@ -166,6 +166,19 @@ def get_current_time():
         """Returns the time as integer number in seconds since the epoch in UTC."""
         return int(time.time())
 
+    @staticmethod
```

Comment:
We decided to avoid any new function to Utility class. This is making this class worse and unusable.

Reply:
This function is not related to specific lr2. It is generic to all.

Reply:
@satish-seagate create a class in mon_utils.py, put existing & this new function under it, & use it,

Reply:
Done